modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,262 @@
+
+* Before this update, when {tekton-chains} was configured to use `gcs` storage, Tekton Chains did not record pipeline run information. With this update, {tekton-chains} records pipeline run information with this storage.
+
+* With this update, performance metrics are available for {tekton-chains}. To access the metrics, expose the `tekton-chains-metrics` service and then use the `/metrics` path on this service, port 9090. These metrics are also available in the {OCP} Monitoring stack.
+
+* With this update, {tekton-chains} uses the new `v2alpha3` record format version when recording pipeline runs and task runs that use the `v1` version value.
+
+* With this update, {tekton-chains} uses the `v1` version of pipeline run and task run formats internally.
+
+[id="tekton-results-new-features-1-14_{context}"]
+=== {tekton-results}
+
+* With this update, if {tekton-results} is installed, {tekton-results} records the summary and record data for pipeline runs started using {pac}.
+
+* With this update, {tekton-results} provides up to 100 megabytes of logging information for a pipeline or task.
+
+* With this update, any authenticated user can view the `tekton-results-api-service` route in the openshift-pipelines namespace to interact with Tekton Results using a REST API.
+
+* With this update, the {tekton-results} API includes a new endpoint for fetching summary and aggregation for a list of records.
+
+* With this update, the `GetLog` endpoint of the {tekton-results} API returns raw bytes with the `text/plain` content type.
+
+With this update, you can optionally specify a custom CA certificate in the `options.configMaps.tekton-results-api-config.data.config.DB_SSLROOTCERT` spec in the `TektonResult` CR. In this case, Tekton Results requires an SSL connection to the database server and uses this certificate for the connection.
+
+[id="breaking-changes-1-14_{context}"]
+== Breaking changes
+
+* With this update, when using the Bundles resolver, you can no longer specify the `serviceAccount` parameter. Instead, you can specify the `secret` parameter to provide the name of a secret containing authentication information for the registry. You must update any tasks or pipelines that use the `serviceAccount` parameter of the Bundles resolver to use the `secret` parameter instead. The `pipeline.bundles-resolver-config.default-service-account` spec in the `TektonConfig` CR is no longer supported.
+
+[id="known-issues-1-14_{context}"]
+== Known issues
+
+* The `tkn pipeline logs` command does not display the logs of tasks that were defined in a pipeline with the `retries: X` parameter.
```

Comment:
@piyush-garg the `tkn` command displays the logs but not for the running pipeline. Am I right?

Reply:
Aah yes, it does not show while pipeline is in progress and `-f` is used, as the pipeline gets completed, it works